src/Hooks/addUser.jsx

```diff
@@ -0,0 +1,15 @@
+import userService from '../Services/user';
+import {
+  message,
+} from '../Components/AntDesign';
+import { HTTP_EXCEPTIONS_MESSAGES } from '../Constants';
+
+export default function addUser(username, email, password) {
```

Comment:
Custom hooks should be started with `use` work, see [the react docs](https://reactjs.org/docs/hooks-custom.html#extracting-a-custom-hook:~:text=A%20custom%20Hook%20is%20a%20JavaScript%20function%20whose%20name%20starts%20with%20%E2%80%9Duse%E2%80%9D%20and%20that%20may%20call)
```suggestion
export default function useAddUser(username, email, password) {
```